vllm/envs.py

```diff
@@ -256,6 +257,10 @@ def get_default_config_root():
     "VLLM_ATTENTION_BACKEND":
     lambda: os.getenv("VLLM_ATTENTION_BACKEND", None),
 
+    # If set, vllm will not use flashinfer sampler
```

Comment:
Isn't this the opposite?
```suggestion
    # If set, vllm will use flashinfer sampler
```